analytic-ui-generator/src/main/java/com.wipro.ats.bdre.augen/AnalyticUIGeneratorMain.java

```diff
@@ -0,0 +1,111 @@
+package com.wipro.ats.bdre.augen;
+
+import java.util.Enumeration;
+import java.util.List;
+
+import com.wipro.ats.bdre.BaseStructure;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wipro.ats.bdre.BaseStructure'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AUselessImportCheck)